www/home.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <meta http-equiv="Content-Security-Policy" content="default-src * 'self' data: gap: https://ssl.gstatic.com 'unsafe-eval'; style-src 'self' 'unsafe-inline'; media-src *">
```

Comment:
Content seucrity policy is correct. Hwoever, would modifiy to each according to what it does. Example img-src for image links instead of given them access from the content source. This however is optional as our current policy should be ok.